File: engine/b/src/main/java/net/crazyinvestor/engine_b/dto/response/BithumbWSOrderbookDepthResponseContent.java
package net.crazyinvestor.engine_b.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public final class BithumbWSOrderbookDepthResponseContent {
    private String symbol;
    private String orderType;
    private String price;
    private String quantity;
    private String total;
}

File: engine/a/ticker_producer_engine/src/main/kotlin/net/crazyinvestor/ticker_producer_engine/dto/bithumb/response/BithumbWSOrderbookDepthResponseContent.kt
package net.crazyinvestor.ticker_producer_engine.dto.bithumb.response

class BithumbWSOrderbookDepthResponseContent (
    val symbol: String?,
    val orderType: String?,
    val price: String?,
    val quantity: String?,
    val total: String?
){
    constructor(): this(null, null, null, null, null){}
}